Library/UIButton+HapticFeedback.swift

```diff
@@ -1,15 +0,0 @@
-import UIKit
-
-extension UIButton {
-  public func generateSelectionFeedback() {
-    UIFeedbackGenerator.ksr_selection()
-  }
-
-  public func generateSuccessFeedback() {
```

Comment:
I believe previously only `generateSuccessFeedback` was used through proxy so the other 2 functions were unused